### ISSUE TITLE
Vanilla drop and power drop being bound to the same key causes both to execute simultaneously (1.12.2)

### ISSUE THREAD

**Issue by EliteMeats** [OmegaJak/Power_Drop#6]
When both the vanilla drop keybind and power drop keybind are on the same key, it causes both to activate at the same time. While this specific issue goes away once the vanilla drop is bound to something else, this also means the vanilla drop cannot be used at all (for example, while hovering over items and pressing Q in the inventory). Ideally power drop should still be usable when bound to the same key as vanilla drop; it should just detect if the user is holding the drop key, for example. This bug is present in the mod for 1.12.2; I do not know if it is present in newer versions as well.

![image](https://github.com/OmegaJak/Power_Drop/assets/70586468/7585753b-34b5-4571-b6d6-a2fbb9c3d825)

https://github.com/OmegaJak/Power_Drop/assets/70586468/4fbf9725-4c6d-47de-a4f6-2534ecbbd81b

**Comment by OmegaJak** [comment]
Yeah, I've never had a great answer for how this should interact with the vanilla drop keybind. I'll give that some more thought.

Interestingly, I'm not seeing this same behavior with Minecraft 1.12.2 and the "MC1.12.2-1.1.2" version of Power Drop (with Forge 14.23.5.2859). Are you using the same versions of the mod and forge as I am? I'm curious why I can't seem to reproduce this behavior.

This problem is present on the 1.18-1.20 versions, if bound to the same key (though it drops many items like the vanilla drop does if held).

As a workaround, does binding Power Drop to Shift+Q and leaving vanilla drop on Q solve it for you on 1.12.2?

**Comment by OmegaJak** [comment]
I specifically addressed this issue in the 1.18-1.20 versions (2.0.1, 2.1.2, and 2.2.1), but haven't yet on the 1.12 version. The fix I used on the newer versions won't work on 1.12, so let me know if the workaround above is sufficient.

**Comment by EliteMeats** [comment]
Hi, sorry for the late response! 

> Interestingly, I'm not seeing this same behavior with Minecraft 1.12.2 and the "MC1.12.2-1.1.2" version of Power Drop (with Forge 14.23.5.2859). Are you using the same versions of the mod and forge as I am? I'm curious why I can't seem to reproduce this behavior.

In the video above I was using version "MC1.12.2-1.1.2", but was using Forge 14.23.5.2860. Not sure if that would somehow affect it, but yeah.

> As a workaround, does binding Power Drop to Shift+Q and leaving vanilla drop on Q solve it for you on 1.12.2?

It does; I bound it to Alt+Q for my purposes, but it's the same thing functionally, so I guess that actually sort of fixes the issue. I  had no idea you could bind things to two keys; would've been very helpful to know that before lol.

I honestly wasn't expecting such a fast resolution, since a lot of modders abandon their 1.12.2 versions, so thanks again for jumping on this so quickly!
